src/components/Button/button.st.css

```diff
@@ -0,0 +1,11 @@
+.root {
+  background: pink;
+  color: #fff;
+  padding: 8px 16px;
```

Comment:
Boilerplate some of the more essential vars and maybe a few code samples of the fancy things that Styable can do. More for copy and paste examples than anything else? Maybe get it in first, we can extend documentation in due course.

Reply:
Yes, this is basically getting Stylable in with Gatsby and a way to test things. We can expand later on.